internal/manifests/collector/container.go

```diff
@@ -107,6 +107,7 @@ func Container(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 	if len(otelcol.Spec.ConfigMaps) > 0 {
 		for keyCfgMap := range otelcol.Spec.ConfigMaps {
 			volumeMounts = append(volumeMounts, corev1.VolumeMount{
+				Name:      "configmap-" + otelcol.Spec.ConfigMaps[keyCfgMap].Name,
```

Comment:
should we add this to the naming package?

Reply:
As we are using to name the default ConfigMaps, I would say to put it together.